sentry.client.config.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "",
+
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 1,
+
+  // Setting this option to true will print useful information to the console while you're setting up Sentry.
+  debug: true,
```

Comment:
Perhaps this should be something like this? (Same for the other config files.)

```suggestion
  debug: process.env.NODE_ENV === "development",
```

Though maybe it'll result in too much noise in our console, in which case maybe it should just be false?

Reply:
Q: Is it "development" or "dev"?

```sh
git --no-pager grep -n "\.NODE_ENV "

esbuild.js:31:  minify: AppConstants.NODE_ENV !== 'dev',
esbuild.js:32:  sourcemap: AppConstants.NODE_ENV !== 'dev',
next.config.js:19:              `script-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-eval' 'unsafe-inline'" : ''} https://*.googletagmanager.com`,
next.config.js:21:              `connect-src 'self' ${process.env.NODE_ENV === 'development' ? 'webpack://*' : ''} https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com`,
next.config.js:24:              `style-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-inline'" : ''}`,
scripts/send-email-to-unresolved-breach-subscribers.js:125:  if (AppConstants.NODE_ENV !== 'tests') {
src/app.js:27:const isDev = AppConstants.NODE_ENV === 'dev'
src/appConstants.js:60:if (!process.env.SERVER_URL && process.env.NODE_ENV === 'heroku') {
src/appConstants.js:77:export default AppConstants.NODE_ENV === 'test'
src/client/js/analytics.js:7:const debugMode = AppConstants.NODE_ENV !== 'production'
src/middleware/error.js:41:    message: process.env.NODE_ENV !== 'production' ? errMsg : 'Something went wrong', // hide error message when in production
src/middleware/error.js:42:    stack: process.env.NODE_ENV === 'dev' ? err.stack : {} // hide stack when not in dev
src/routes/api/v1/user.js:26:if (AppConstants.NODE_ENV === 'dev') {
src/routes/index.js:45:if (AppConstants.NODE_ENV !== 'production') {
src/routes/user.js:35:if (AppConstants.NODE_ENV === 'dev') {
```

```sh
git --no-pager grep -n "'development'"

next.config.js:19:              `script-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-eval' 'unsafe-inline'" : ''} https://*.googletagmanager.com`,
next.config.js:21:              `connect-src 'self' ${process.env.NODE_ENV === 'development' ? 'webpack://*' : ''} https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com`,
next.config.js:24:              `style-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-inline'" : ''}`,
next.config.js:80:    const noindexEnvs = ['dev', 'development', 'heroku', 'stage']
```

Reply:
Yes I noticed this too, the old app called it `dev` and the nextjs calls it `development`.